msal/src/main/java/com/microsoft/identity/client/internal/controllers/OperationParametersAdapter.java

```diff
@@ -97,6 +97,7 @@ public static AcquireTokenOperationParameters createAcquireTokenOperationParamet
 
         acquireTokenOperationParameters.setAuthenticationScheme(
                 AuthenticationSchemeFactory.createScheme(
+                        acquireTokenParameters.getActivity(),
```

Comment:
Is there a `createScheme` method that takes an `Activity`? I can't find it.

Reply:
`Activity` is a subclass of `Context`

Reply:
Btw I am going to clean up this constructor a little bit to remove the hard reference to the Context and instead include a reference to the `IClockSkewManger` -- will be pushed up momentarily

Reply:
Cleaned up here:
https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/868/commits/dff13326e369b599f0ea4aa483c401431b56076b